vim/plugin-managers.md

```diff
@@ -0,0 +1,26 @@
+# Plugin Managers
+
+Today I learned that vim has two main plugin managers:
+
+1. Tim Pope<sup>1</sup>'s [Pathogen][1]
+1. gmarik's [Vundle][2]
```

Comment:
Markdown doesn't care about what the numbers are. It's just converting numeric values to <ol><li> elements and HTML is doing the ordering. Hey! That could be your TIL :smile: